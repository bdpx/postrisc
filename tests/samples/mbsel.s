.text
    alloc	96
    write	"check mbsel instruction"
    ldi.l	%r6, ((0x3333333333333333 ^ 0x5555555555555555) & 0xff00ff00ff00ff00) ^ 0x5555555555555555
    write	"mbsel: %x64(r6)"
    ldi.l	%r3, 0x3333333333333333
    ldi.l	%r4, 0x5555555555555555
    ldi.l	%r5, 0xff00ff00ff00ff00
    mbsel	%r6, %r3, %r4, %r5
    write	"mbsel: %x64(r6)"

    write	"end_mbsel_test"
.end
