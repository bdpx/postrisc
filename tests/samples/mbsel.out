check mbsel instruction
mbsel: 3355335533553355
mbsel: 3355335533553355
end_mbsel_test
